src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,141 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table = new Node[INITIAL_CAPACITY];
```

Comment:
all not static fields should be initialized it in the constuctor
```suggestion
    private Node<K, V>[] table ;
```